File: PharmacyApp/src/main/java/com/pharmacy/dto/ObjectionDto.java
package com.pharmacy.dto;

public class ObjectionDto {

	private String content;
	private String idEncoded;
	
	public ObjectionDto(String content, String idEncoded) {
		super();
		this.content = content;
		this.idEncoded = idEncoded;
	}
	
	public ObjectionDto() {
		
	}

	public String getContent() {
		return content;
	}

	public void setContent(String content) {
		this.content = content;
	}

	public String getIdEncoded() {
		return idEncoded;
	}

	public void setIdEncoded(String idEncoded) {
		this.idEncoded = idEncoded;
	}
	
	

	
	
}

File: PharmacyApplication/PharmacyApplication/Dto/ObjectionDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PharmacyAPI.Dto
{
    public class ObjectionDto
    {
        public string Id { get; set; }
        public string Content { get; set; }

        public ObjectionDto() { }

        public ObjectionDto(string content, string idEncoded)
        {
            Content = content;
            Id = idEncoded;
        }
    }
}
